bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -109,13 +155,15 @@ private void initSqlTypes() {
         sqlTypes.put("CALLITEM", "VARCHAR(200)");
         sqlTypes.put("COLORITEM", "VARCHAR(70)");
         sqlTypes.put("CONTACTITEM", "VARCHAR(6)");
-        sqlTypes.put("DATETIMEITEM", "DATETIME");
+        sqlTypes.put("DATETIMEITEM", "TIMESTAMP");
```

Comment:
I got problems using TIMESTAMP and DATETIME mixed. With TIMESTAMP I get better Results now. For me it was crucial that a TIMESTAMP is used with Time zones conversation.

After some investigation, there are pros and cons:
http://cdn.oreillystatic.com/en/assets/1/event/36/Time%20Zones%20and%20MySQL%20Presentation.pdf